extension/chrome/elements/pgp_block.ts

```diff
@@ -378,8 +378,10 @@ Catch.try(async () => {
         isHtml = true;
       } else if (typeof decoded.text !== 'undefined') {
         decryptedContent = decoded.text;
+      } else {
+        decryptedContent = '';
       }
-      if (decoded.subject) {
+      if (decoded.subject && isEncrypted) {
```

Comment:
Here I have fixed unrelated issue, it does not make sense to show `Encrypted Subject` if this was a signed message.